src/main/java/core/basesyntax/service/ActionStrategyImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.action.ActionHandler;
+import java.util.Map;
+
+public class ActionStrategyImpl implements ActionStrategy {
+    private Map<Operation, ActionHandler> actionHandlerMap;
+    private ActionStrategy strategy;
+
+    public ActionStrategyImpl(Map<Operation, ActionHandler> actionHandlerMap) {
+        this.actionHandlerMap = actionHandlerMap;
+    }
+
+    public ActionStrategyImpl(ActionStrategy strategy) {
+        this.strategy = strategy;
+    }
+
+    public ActionStrategyImpl() {
+    }
```

Comment:
remove those two constructors, rewrite the code in main to use only the first one